extensions/amp-subscriptions/0.1/test/test-platform-store.js

```diff
@@ -257,16 +256,11 @@ describes.realWin('Platform store', {}, () => {
   });
 
   describe('reportPlatformFailure_', () => {
-    let errorSpy;
-    beforeEach(() => {
-      errorSpy = sandbox.spy(user(), 'error');
-    });
 
-    // TODO(prateekbh, #14336): Fails due to console errors.
-    it.skip('should report fatal error if all platforms fail', () => {
+    it('should report fatal error if all platforms fail', () => {
       platformStore.reportPlatformFailure('service1');
-      platformStore.reportPlatformFailure('service2');
-      expect(errorSpy).to.be.calledOnce;
+      expect(() => platformStore.reportPlatformFailure('service2'))
+          .to.throw(/All platforms have failed to resolve/);
```

Comment:
@prateekbh FYI, this test can now be fixed more easily using `allowConsoleError`. I've reverted this change and applied the simple fix in an upcoming change.